### PR TITLE
Authors

### PR DESCRIPTION
**Pull request by scopatz**
A new authors activity. Should help with getting conda-build on `rever`.

### CODE REVIEW

**Comment by scopatz** [comment]
CC @regro/auto-tick @CJ-Wright

**Comment by asmeurer** [comment]
You may be interested in the scripts SymPy uses for these, if you haven't seen them already. https://github.com/sympy/sympy/blob/master/bin/authors_update.py https://github.com/sympy/sympy/blob/master/bin/mailmap_update.py

**Comment by scopatz** [comment]
@CJ-Wright - added docs!

@asmeurer - yeah thanks for the tip! I actually looked at those while writing this for help with some of the ideas here, esp. mailmap.

File: docs/index.rst [hunk 1]
```diff
@@ -71,6 +71,8 @@ Contents
 
     tutorial
     usepatterns
+    news
```
Comment (CJ-Wright) [inline docs/index.rst]: Nice catch 🐑

**Pull request merged**